wear/src/main/java/io/homeassistant/companion/android/sensors/SensorReceiver.kt

```diff
@@ -0,0 +1,71 @@
+package io.homeassistant.companion.android.sensors
+
+import android.annotation.SuppressLint
+import dagger.hilt.android.AndroidEntryPoint
+import io.homeassistant.companion.android.BuildConfig
+import io.homeassistant.companion.android.common.sensors.BatterySensorManager
+import io.homeassistant.companion.android.common.sensors.SensorManager
+import io.homeassistant.companion.android.common.sensors.SensorReceiverBase
+
+@AndroidEntryPoint
+class SensorReceiver : SensorReceiverBase() {
+
+    override val tag: String
+        get() = TAG
+
+    override val currentAppVersion: String
+        get() = BuildConfig.VERSION_NAME
+
+    override val managers: List<SensorManager>
+        get() = MANAGERS
+
+    companion object {
+        const val TAG = "SensorReceiver"
+        val MANAGERS = listOf(
```

Comment:
For now lets remove these rather than comment out.

Reply:
Done